.github/actions/spell-check/dictionary/dictionary.txt

```diff
@@ -55872,6 +55872,7 @@ burly
 burly-boned
 burly-faced
 burly-headed
+burlywood
```

Comment:
Do you still need the additions you made here now that there's a separate colors.txt dictionary?

Reply:
That’s inspiring. Thanks. I’ll clean up this later.